packages/components/psammead-section-label/src/index.jsx

```diff
@@ -72,68 +61,79 @@ const Wrapper = styled.div`
     `}
 `;
 
-Wrapper.propTypes = {
+SectionLabelWrapper.propTypes = {
   bar: bool.isRequired,
   script: shape(scriptPropType).isRequired,
   visuallyHidden: bool.isRequired,
 };
 
-const paddingDir = ({ dir }) => `padding-${dir === 'ltr' ? 'right' : 'left'}`;
-
-const Title = styled.h2`
-  ${({ script }) => script && getDoublePica(script)};
-  color: ${C_EBON};
-  background-color: ${C_WHITE};
-  ${({ service }) => getSansRegular(service)}
-  display: inline-block;
-  position: relative;
-
-  /* Unset the browser's default margins. */
+const Heading = styled.h2`
+  /* reset default margins */
   margin: 0;
-
-  ${paddingDir}: ${GEL_SPACING};
-
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    ${paddingDir}: ${GEL_SPACING_DBL};
-  }
+  padding: 0;
 `;
 
-Title.propTypes = {
-  dir: oneOf(['ltr', 'rtl']).isRequired,
-  script: shape(scriptPropType).isRequired,
-  service: string.isRequired,
-};
-
 const SectionLabel = ({
   bar,
   children: title,
   dir,
+  href,
   labelId,
+  linkText,
   script,
-  visuallyHidden,
   service,
+  visuallyHidden,
 }) => (
-  <Wrapper script={script} bar={bar} visuallyHidden={visuallyHidden}>
-    <Title script={script} dir={dir} id={labelId} service={service}>
-      {title}
-    </Title>
-  </Wrapper>
+  <SectionLabelWrapper
```

Comment:
Do we need a region landmark here?

Reply:
I don't believe so, the region landmark should be set by a `<section>` which wraps the entire section (which this component does not do). That `<section>` lives in simorgh, and passes through the `labelId` for this component to use, which allows eg. `<section role="region" aria-labelledby={labelId}>`